common/assets/styles/etherwallet-fonts.less

```diff
@@ -1,24 +1,13 @@
 @font-face {
   font-family: 'Lato';
-  src: url('../fonts/Lato-Light.woff') format('woff'),
-    url('../fonts/Lato-Light.ttf') format('truetype');
+  src: url('../fonts/Lato-Light.woff2') format('woff2');
   font-style: normal;
   font-weight: 300;
 }
 
 @font-face {
   font-family: 'Lato';
-  src: url('../fonts/Lato-Regular.woff') format('woff'),
-    url('../fonts/Lato-Regular.ttf') format('truetype');
+  src: url('../fonts/Lato-Regular.woff2') format('woff2');
```

Comment:
[Not all browsers are up to woff2 yet](https://caniuse.com/#feat=woff2), if we're going to narrow down to one I'd stick with woff.

Reply:
Good catch, yea it'd be better to support woff then